eng/pipeline/stages/go-builder-matrix-stages.yml

```diff
@@ -81,6 +81,13 @@ stages:
             - { os: linux, arch: arm64, config: buildandpack }
         - ${{ if parameters.innerloop }}:
           - { os: darwin, arch: amd64, config: devscript }
+          - { os: darwin, arch: amd64, config: test }
+          - { experiment: darwincrypto, os: darwin, arch: amd64, config: test }
+          - { experiment: darwincrypto, os: darwin, arch: amd64, config: test, fips: true }
+          # - { os: darwin, arch: arm64, config: devscript }
```

Comment:
noting that I'll enable this in a follow-up PR once a new version of GoTool@0 has been released that contains https://github.com/microsoft/azure-pipelines-tasks/pull/20783